src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -98,11 +99,13 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
           'press_release',
         );
         break;
-      case 'health_services_listing':
-        pageCompiled.clinicalHealthServices = sortServices(
-          pageCompiled.fieldOffice.entity.reverseFieldRegionPageNode.entities,
-        );
-        break;
+      // case 'health_services_listing':
+      //   if (pageCompiled.fieldOffice !== null) {
+      //     pageCompiled.clinicalHealthServices = sortServices(
+      //       pageCompiled.fieldOffice.entity.reverseFieldRegionPageNode.entities,
+      //     );
+      //   }
+      //   break;
```

Comment:
Why was this removed? Without sorting, do the services still appear in alphabetical order on the page (e.g. /pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive)?

Reply:
just failing tests, we wouldn't merge this part